client/src/functions/SelectFiveRandomFromList.js

```diff
@@ -0,0 +1,8 @@
+export default function SelectFiveRandomFromList(listOfSongs) {
+    const randomList = [];
+    for (let i = 0; i < 5; i++) {
+        const randomNum = Math.floor(Math.random() * 5);
```

Comment:
Question: Does this select duplicates and push them to randomList? i.e. When you were young gets selected by random twice :)

Reply:
Thanks for pointing this out, I overlooked it last night. I have refactored the code to include an if statement to check if array has the song/string before adding it to the array